proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -30,39 +30,56 @@
     return;
   }
 
-  var enableAuthorizationLookup = context.getVariable("app.enable-authorization-lookup");
+  var nrlPermissions = context.getVariable("app.nrl-permissions");
+  var hasAllPointerPermissions = false;
 
+  if (nrlPermissions != null) {
+    // Convert it into a complex object
+    var permissionLines = nrlPermissions.split(/\s+/);
+    var permissions = [];
+    for (var i = 0; i < permissionLines.length; i++) {
+      var permissionLine = permissionLines[i];
+      if (permissionLine && permissionLine.trim().length !== 0) {
+        permissions.push(permissionLine);
+      }
+    }
+    if (nrlPermissions.includes("allow-all-pointer-types") === true) {
+      hasAllPointerPermissions = true;
+    }
+    nrlPermissions = permissions;
+  }
+
+  var enableAuthorizationLookup = context.getVariable("app.enable-authorization-lookup");
   if(enableAuthorizationLookup == "true") {
     enableAuthorizationLookup = true
-  } else if (enableAuthorizationLookup === null) {
+  } else if (enableAuthorizationLookup === null  || enableAuthorizationLookup == "false") {
```

Comment:
This fix needs to be applied to the consumer proxy as well.

Reply:
yes

Reply:
https://github.com/NHSDigital/nrl-consumer-api/pull/126

Reply:
if you would kindly review